rest/rest-cli/src/main/java/org/ow2/proactive_grid_cloud_portal/cli/cmd/rm/ListNodeCommand.java

```diff
@@ -76,8 +75,18 @@ public void execute(ApplicationContext currentContext) throws CLIException {
                     selectedNodeEvents = selectedList.toArray(new NodeEventView[selectedList.size()]);
                 }
             }
-            resultStack(currentContext).push(selectedNodeEvents);
-            writeLine(currentContext, "%s", StringUtility.string(selectedNodeEvents));
+            List<NodeEventView> filtered = new ArrayList<>(selectedNodeEvents.length);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') A "NullPointerException" could be thrown; "selectedNodeEvents" is nullable here. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2259)